src/main/flight/pid.c

```diff
@@ -445,19 +446,63 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #ifdef USE_GPS_RESCUE
     angle += gpsRescueAngle[axis] / 100; // ANGLE IS IN CENTIDEGREES
 #endif
-    angle = constrainf(angle, -levelAngleLimit, levelAngleLimit);
```

Comment:
The `angle` variable is actually the Target angle, as derived from sticks.  

Maybe the code would be more clear if it was re-named `angleTarget`, since `angle` could be any angle, otherwise.